kitsune/community/signals.py

```diff
@@ -8,12 +8,28 @@
 from kitsune.wiki.models import Document, Revision
 
 
+def is_marked_for_deletion(user, origin):
```

Comment:
I would change the name to something that denotes that we are checking for user deletion (eg `skip_on_user_deletion`)